hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/hudi/DataSourceOptions.scala

```diff
@@ -108,7 +108,7 @@ object DataSourceReadOptions {
     .key("hoodie.datasource.read.begin.instanttime")
     .noDefaultValue()
     .withDocumentation("Required when `" + QUERY_TYPE.key() + "` is set to `" + QUERY_TYPE_INCREMENTAL_OPT_VAL + "`. Represents the instant time to start incrementally pulling data from. The instanttime here need not necessarily "
-      + "correspond to an instant on the timeline. New data written with an instant_time > BEGIN_INSTANTTIME are fetched out. "
+      + "correspond to an instant on the timeline. New data written with an instant_time >= BEGIN_INSTANTTIME are fetched out. "
```

Comment:
with completion_time ?

Reply:
Fixed.  I also renamed the variable and revised the docs as a whole.